3D Chess/Assets/Scripts/PieceSpawner.cs

```diff
@@ -0,0 +1,104 @@
+using UnityEngine;
+
+public class PieceSpawner : MonoBehaviour
+{
+    [SerializeField] Piece prefabPawnWhite = default;
+    [SerializeField] Piece prefabRookWhite = default;
+    [SerializeField] Piece prefabKnightWhite = default;
+    [SerializeField] Piece prefabBishopWhite = default;
+    [SerializeField] Piece prefabQueenWhite = default;
+    [SerializeField] Piece prefabKingWhite = default;
+    [SerializeField] Piece prefabPawnBlack = default;
+    [SerializeField] Piece prefabRookBlack = default;
+    [SerializeField] Piece prefabKnightBlack = default;
+    [SerializeField] Piece prefabBishopBlack = default;
+    [SerializeField] Piece prefabQueenBlack = default;
+    [SerializeField] Piece prefabKingBlack = default;
```

Comment:
Depending on how your warnings are configured you would see a `Field is never assigned to` warning since it actually is never assigned to in this class. This happens in the Unity editor but Visual Studio cannot check that at this point and has to show the warning.
Setting it to default silences this warning. I guess a pragma would be the other option but in every Unity thread I've ever seen about this warning everyone goes the default value route (either `default` or actually setting the default value itself).

There are, however, access modifiers missing.
Added with b7aa778, should be consistent with everything else now.